app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -241,8 +243,32 @@ public void onResume() {
             _hlEditor.setText(CoolExperimentalStuff.convertEpubToText(_document.getFile(), getString(R.string.page)));
         }
 
-        // Set initial wrap state
         initDocState();
+
+        // Run once in first onResume if visible. Should always be visible here
+        if (_firstStart && getUserVisibleHint()) {
+            _firstStart = false;
+            // 1. Start on bottom if that is the setting or if quicknote file
+            // 2. Do not set focus or set cursor for todo file
+            // 3. If other file and save cursor position exists, start there
```

Comment:
Cleaned up the logic here. @gsantner Please review and let me know if this behavior is good. It is the same as what exists currently, except for quicknote, which always starts at the bottom now.